### ISSUE TITLE
[BUG] Sidebar Search Over Menu Items [sidebar-custom-search] redirect to 404

### ISSUE THREAD

**Issue by AMoktar** [jeroennoten/Laravel-AdminLTE#1044]
Hello, 
first thanks a lot for this great work
I'm using sidebar-custom-search
```
 [
            'type' => 'sidebar-menu-search',
            'text' => 'search',
        ],
````
when I navigate to results , it goes to something like http://127.0.0.1:8000/admin/http%3A//127.0.0.1%3A8000/admin/users

**Comment by Shidersz** [comment]
This is a bug from the underlying **AdminLTE v3.1.0** package. There is a temporal solution on #900 and this problem will be solved on the next release of **AdminLTE v3.2.0**.

**Comment by AMoktar** [comment]
Thanks @Shidersz 
this is my release : "jeroennoten/laravel-adminlte": "^3.7"
Do I need to update  ?
